desktop/libs/azure/src/azure/abfs/upload.py

```diff
@@ -35,10 +35,89 @@
 else:
   from django.utils.translation import ugettext as _
 
-DEFAULT_WRITE_SIZE = 30 * 1000 * 1000 # TODO: set in configuration
+DEFAULT_WRITE_SIZE = 128 * 1000 * 1000 # TODO: set in configuration
 
 LOG = logging.getLogger()
 
+import io
+import os
+from desktop.lib.exceptions_renderable import PopupException
+from filebrowser.conf import ARCHIVE_UPLOAD_TEMPDIR
+from filebrowser.utils import generate_chunks
+
+class ABFSFineUploaderChunkedUpload(object):
```

Comment:
Same comments for Popupexception and Log messaged and better variable naming just like for new OFS handler above.